### PR TITLE
Clean up NIC docs for better wording

### PR DESCRIPTION
**Pull request by sjberman**
Slight adjustment to the wording of the NIC doc so that a user knows that it's informational.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/nginx-service-mesh/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto main
- [x] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

**Review by chase-kiefer — APPROVED** [review]
Good addition. LGTM.

File: docs/content/tutorials/kic/deploy-with-kic.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ docs: "DOCS-721"
 You can deploy NGINX Ingress Controller for Kubernetes with NGINX Service Mesh to control both ingress and [egress](#enable-egress) traffic.
 
 {{< important >}}
-There are two versions of NGINX Ingress Controller for Kubernetes: NGINX Open Source and NGINX Plus.
+NGINX Ingress Controller can be used free with NGINX Open Source and paying customers have access to NGINX Ingress Controller with NGINX Plus.
```
Comment (jbyers19) [inline docs/content/tutorials/kic/deploy-with-kic.md:16]:
```suggestion
NGINX Ingress Controller can be used for free with NGINX Open Source. Paying customers have access to NGINX Ingress Controller with NGINX Plus.
```

File: docs/content/tutorials/kic/egress-walkthrough.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ docs: "DOCS-722"
 
 Learn how to create internal routes in NGINX Ingress Controller to securely route egress traffic to non-meshed services. 
 {{< note >}}
-There are two versions of NGINX Ingress Controller for Kubernetes: NGINX Open Source and NGINX Plus.
+NGINX Ingress Controller can be used free with NGINX Open Source and paying customers have access to NGINX Ingress Controller with NGINX Plus.
```
Comment (jbyers19) [inline docs/content/tutorials/kic/egress-walkthrough.md:14]:
```suggestion
NGINX Ingress Controller can be used for free with NGINX Open Source. Paying customers have access to NGINX Ingress Controller with NGINX Plus.
```

File: docs/content/tutorials/kic/ingress-udp-walkthrough.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ Objectives:
 - Create a Kubernetes TransportServer resource for the udp-listener application.
 
 {{< note >}}
-There are two versions of NGINX Ingress Controller for Kubernetes: NGINX Open Source and NGINX Plus.
+NGINX Ingress Controller can be used free with NGINX Open Source and paying customers have access to NGINX Ingress Controller with NGINX Plus.
```
Comment (jbyers19) [inline docs/content/tutorials/kic/ingress-udp-walkthrough.md:24]:
```suggestion
NGINX Ingress Controller can be used for free with NGINX Open Source. Paying customers have access to NGINX Ingress Controller with NGINX Plus.
```

File: docs/content/tutorials/kic/ingress-walkthrough.md [hunk 4]
```diff
@@ -24,7 +24,7 @@ All communication between the NGINX Ingress Controller and the Bookinfo applicat
 {{< /note >}}
 
 {{< note >}}
-There are two versions of NGINX Ingress Controller for Kubernetes: NGINX Open Source and NGINX Plus.
+NGINX Ingress Controller can be used free with NGINX Open Source and paying customers have access to NGINX Ingress Controller with NGINX Plus.
```
Comment (jbyers19) [inline docs/content/tutorials/kic/ingress-walkthrough.md:27]:
```suggestion
NGINX Ingress Controller can be used for free with NGINX Open Source. Paying customers have access to NGINX Ingress Controller with NGINX Plus.
```

**Review by jbyers19 — APPROVED** [review]
Left some nit suggestions, but it looks fine to me either way.

**Pull request merged**